Dockerfile

```diff
@@ -1,21 +1,23 @@
-# syntax=docker/dockerfile:1
-FROM mcr.microsoft.com/devcontainers/javascript-node:0-18 AS build
-ARG REACT_APP_BACKEND_URL
+FROM node:18.16.0-alpine AS build
+
 WORKDIR /code
+
+ARG REACT_APP_BACKEND_URL
+ARG REACT_APP_DEBUG_MODE
+ARG REACT_APP_GITHUB_API_URL
+ARG REACT_APP_SEARCH_SOURCE
```

Comment:
you switched ENV to ARG. ARG is good because it forces the builder to have values for BUILD, but they should also be set as ENV as it seems there is an intent to use this container to build the source again again.

Reply:
Done